src/token/src/ledger.rs

```diff
@@ -41,7 +42,13 @@ impl Ledger {
             .iter()
             .rev()
             .filter(|tx| who.map_or(true, |c| c == tx.from || c == tx.to || Some(c) == tx.caller))
-            .filter(|tx| transaction_id.map_or(true, |id| id >= tx.index))
```

Comment:
should `=` in `id >= tx.index` be removed then?

Reply:
I believe that is old code

Reply:
Oh I see , it has been fixed.